docs/developers-clear-containers-install.md

```diff
@@ -124,13 +125,43 @@ See [the upgrading document](upgrading.md) for further details.
    $ sudo sed -i.bak -e 's!^\(image = ".*"\)!# \1 \
    image = "/usr/share/clear-containers/container.img"!g' \
    /usr/share/defaults/clear-containers/configuration.toml
-   
+   ```
 For more details on the runtime's build system, run:
```

Comment:
hmm, this should be indented along with the code block.

Reply:
it is indented  with the code block above. It was missing to close it.